portfolio/src/main/webapp/index.html

```diff
@@ -11,13 +11,27 @@
       <h1>Ava Garcia's Portfolio</h1>
       <p>
       My name is Ava Garcia and I am a rising junior at Carnegie Mellon who is
-      majoring in artificial intelligence. My interests include computer
-      science, social choice theory, and video games. I am currently an intern
-      at Google, which is the same company that my father works for.
+      majoring in artificial intelligence. I am currently working as an intern
+      at Google, which is the same company that my father works for. Last year I
+      worked on implementing a system that allows technicians to manage their
+      workflows when repairing and replacing machines in Google's datacenters.
+      This year, all my work will be open source and can be found at
+      <a href="https://github.com/avagarcia0">my GitHub</a>. My interests
+      include computer science, video games, and social choice theory.
       </p>
-      <img class="resize" src=/images/VSE.jpg alt="Voter Satisfaction Efficiency Graph">
-      <p>This is my portfolio.</p>
-      <p>Click here to get the name of a random voting method:</p>
+      <p>
+      One cool concept in social choice theory is Voter Satisfaction Efficiency,
+      or VSE for short. VSE is intended to act as a holistic measure of how well
+      a voting method does at satisfying voter's preferences. However, because
+      it is hard to get data from real-life elections and even harder to read
+      people's minds to get their preferences, VSE is measured through simulated
+      elections. The following is a graphic showing VSE results for a set of
+      simulations conducted by Jameson Quinn:
+      </p>
```

Comment:
Removing the </p> tag before the button changes the layout of the page so that there's no space between the button and the paragraph.

Before: https://screenshot.googleplex.com/h1ZJNh8sXS7
After: https://screenshot.googleplex.com/DZH5gr0P6P3

Reply:
As discussed button is an inline element, so we could keep the end tags or create a new <p> for the button if the space is desired.